tests/src/kernel/hipLaunchParm.cpp

```diff
@@ -25,11 +25,140 @@ THE SOFTWARE.
 
 #include "hip/hip_runtime.h"
 #include "test_common.h"
-#include "hip/hip_runtime_api.h"
-#include <iostream>
 
-__global__ void vAdd(hipLaunchParm lp, float* a) {}
+#define DISABLE_TEST 0
+
+static const int  BLOCK_DIM_SIZE = 1024;
+
+// This test is to verify Struct with variables to check the hipLaunchKernel() support, read and write into the same struct
```

Comment:
should remove the comment "read and write into the same structure" as we only read from the struct passed by value. same as below.

Reply:
Done